examples/webgl_mesh_batch.html

```diff
@@ -104,7 +104,7 @@
 			geometries = [
 				new THREE.ConeGeometry( 1.0, 2.0 ),
 				new THREE.BoxGeometry( 2.0, 2.0, 2.0 ),
-				new THREE.SphereGeometry( 1.0 ),
+				new THREE.SphereGeometry( 1.0, 16, 8 ),
```

Comment:
With the errors I found that we were overwriting the end of BatchedMesh geometry index in the example - so I simplified the sphere so it has fewer vertices to avoid that case.